server/plugin.go

```diff
@@ -456,7 +456,7 @@ func (p *Plugin) syncUsers() {
 					}
 				} else {
 					// Skip syncing of MS Teams guest user.
-					p.API.LogDebug("Skipping syncing of the guest user", "MMUserID", mmUser.Id, "TeamsUserID", msUser.ID)
```

Comment:
Thanks for catching this! Looks like I missed that `isUserPresent` might be `false` altogether here leaving `mmUser` as `nil`.